File: pset1/cash/cash.c
#include<stdio.h>
#include<math.h>
#include<cs50.h>
int main(void)
{
    float dollars;
    //prompt user for change owed
    do
    {
        dollars = get_float("Change owed: ");
    }
    while (dollars < 0);

    int cents = round(dollars * 100);
    int coins = 0;
    int denominations[] = {25, 10, 5, 1};
    int size = 4;

    for (int i = 0; i < size; i++)
    {
        // divide cents by the denominations and increment coins with the result
        coins += cents / denominations[i];
        // allocate the remainder to cents
        cents %= denominations [i];
    }
    printf("%i\n", coins);
}

File: pset6/cash/cash.py
from cs50 import get_float

while True:
    # prompt the user to enter chage owed dollars. User must enter values < 0
    dollars = get_float("Change owed: ")
    if dollars >= 0:
        break
# convert dollars to cents

cents = int((dollars * 100))

coins = 0
for coin in [25, 10, 5, 1]:
    coins += cents // coin
    cents %= coin

print(coins)